psql2mysql.py

```diff
@@ -176,19 +180,22 @@ def do_prechecks(config):
     db.connect()
     tables = db.getSortedTables()
     for table in tables:
-        rows = db.scanTablefor4ByteUtf8Char(table)
-        if rows:
-            print("Table '%s' contains 4 Byte UTF8 characters, which are, "
+        incompatibles = db.scanTablefor4ByteUtf8Char(table)
+        if incompatibles:
+            print("Table '%s' contains 4 Byte UTF8 characters which are "
                   "incompatible with the 'utf8' encoding used by MariaDB" %
                   table.name
             )
             print("The following rows are affected.")
-            x = PrettyTable()
-            x.field_names = ["Affected Column"] + list(table.columns)
-            for row in rows:
-                x.add_row([row[0]] + list(row[1]))
-            print x
-            raise Exception("UTF8 mb4 found")
+            last_id = ""
+            for item in incompatibles:
+                item_id = ','.join(item["primary"])
+                if last_id != item_id:
+                    print item_id
+                last_id = item_id
+                print "\t Column: ", item["column"]
+                print "\t Value: ", item["value"]
```

Comment:
I certainly agree with reducing the printed columns here to just small subset. But what was wrong with the table output? (I am also not a huge fan of it, but I think the new format is less appealing)

Are you ok with it I restore the prettytable output (with a reduced list of columns) for now?

Reply:
OK, I did not look what prettytable could show. To me it looked like a huge amount of uninteresting texts

Reply:
>  To me it looked like a huge amount of uninteresting texts
Yes the original output was too much. I've just push an additional commit. Output looks like this now: (Do you like that better?)

```
+-----------------------------------------+---------------------+-------+
|               Primary Key               |   Affected Column   | Value |
+-----------------------------------------+---------------------+-------+
| id=9dca024e-21ce-46ec-828d-0126fcac0f31 |     display_name    |   💩   |
| id=30dbb809-8706-4a3c-a652-f1c0f671f0a4 | display_description |   💬   |
| id=ae6ff387-75f8-45da-bd5f-ee47acc6d225 |     display_name    |   e💬  |
| id=d293b25b-6ba3-4549-9fc6-5d3ec39d0659 | display_description |   💪   |
+-----------------------------------------+---------------------+-------+
```

Reply:
There seems to be a formatting error however with those UTF character. (Hm, how I love it :()

Feel free to remove that commit if you don't like it. It's not that important.

Reply:
This table is really nicer than the one before, I do not really mind that tiny formatting error